apps/charterafrica/src/pages/index.page.js

```diff
@@ -46,6 +46,21 @@ export async function getStaticProps({ defaultLocale, locale, locales }) {
     fallbackLocale: defaultLocale,
   });
 
+  const {
+    siteDescription = [{ children: [{ text: null }] }],
+    projectDescription = [{ children: [{ text: null }] }],
+    contact = { email: null },
+    copyright = null,
+    links = null,
+    footerLogo = null,
+  } = await payload.findGlobal("footer", {
+    locale,
+    fallbackLocale: defaultLocale,
+  });
+
+  const [siteDesc] = siteDescription;
+  const [projectDesc] = projectDescription;
```

Comment:
Why do we need all this jumping around? if Footer exists in the CMS, all the above fields are required, no?